packages/fxa-settings/src/components/PageAvatar/buttons.tsx

```diff
@@ -18,7 +18,7 @@ import { ReactComponent as ZoomInIcon } from './zoom-in.svg';
 import { ReactComponent as RotateIcon } from './rotate.svg';
 
 const buttonClass = `mx-2 text-grey-500
-hover:text-grey-600 hover:text-grey-600
+hover:text-grey-900 hover:text-grey-900
```

Comment:
Not really part of your PR but `hover:text-grey-900` is on here twice. 🤔

Reply:
Interesting, I will clean this up anyway.